src/main/resources/RecruitmentApp/Configuration.xml

```diff
@@ -31,22 +31,44 @@
 #end
 == Users ==
 {{html}}
-    Specify the space from where to get assignee users: 
-    &lt;form method="post" action=""&gt;
-        &lt;input type="text" name="userSpace" value="$userConfigObj.wiki" /&gt;
+  &lt;form method="post" class="xform" action=""&gt;
+    &lt;dl&gt;
+      &lt;dt&gt;
+        &lt;label for="userSpace"&gt;Users page&lt;/label&gt;
+        &lt;span class="xHint"&gt;Page from where to get assignee users&lt;/span&gt;
+      &lt;/dt&gt;
+      &lt;dd&gt;
+        &lt;input type="text" id="userSpace" name="userSpace" value="$userConfigObj.wiki" /&gt;
+      &lt;/dd&gt;
+    &lt;/dl&gt;
+    &lt;p&gt;
+      &lt;span class="buttonwrapper"&gt;
         &lt;input type="submit" value="Update" class="button" /&gt;

```

Comment:
Also for buttons.... See if you can reuse some values from XWiki.

Reply:
@evalica Where can I find values for common translations? I tried looking but couldn't find any.
Also if there are no common translations, do I put new keys for button values since they are supposed to support translations.
And is `recruitment.config.<propertyName>` fine for the key name?
Or is `recruitment.admin.<propertyName>` better.

Reply:
Generic translations can be found in https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties#L115 

App translations at https://github.com/xwiki-contrib/application-recruitment/blob/master/src/main/resources/RecruitmentApp/RecruitmentAppTranslations.xml